gaia-header.js

```diff
@@ -401,31 +758,52 @@ module.exports = Component.register('gaia-header', {
       var(--gaia-header-button-color));
   }
 
-  /** Icons
-   ---------------------------------------------------------*/
+  /**
+   * If users want their action button
+   * to be in the component's light-dom
+   * they can add an .action class
+   * to make it look like the
+   * shadow action button.
+   */
 
-  [class^="icon-"]:before,
-  [class*="icon-"]:before {
-    font-family: 'gaia-icons';
-    font-style: normal;
-    text-rendering: optimizeLegibility;
-    font-weight: 500;
+  ::content .action {
+    color:
+      var(--header-action-button-color,
+      var(--header-icon-color,
+      var(--gaia-header-button-color)));
   }
 
-  .icon-menu:before { content: 'menu'; }
-  .icon-close:before { content: 'close'; }
-  .icon-back:before { content: 'back'; }
+  /**
+   * [data-icon]:empty
+   *
+   * Icon buttons with no textContent,
+   * should always be 50px.
+   *
+   * This is to prevent buttons being
+   * larger than they should be before
+   * icon-font has loaded.
+   */
 
-  </style>
+  ::content [data-icon]:empty {
+    width: 50px;
+  }
 
-  <div class="inner">
-    <button class="action-button">
-      <content select=".l10n-action"></content>
-    </button>
-    <content select="h1,h2,h3,h4,a,button"></content>
-  </div>`
+  </style>`,
+
+  // Test hooks
```

Comment:
It kind of a artifact from older version. The only reason we keep it on the proto is so that we can stub it in testing.